README.adoc

```diff
@@ -1860,6 +1860,19 @@ Time.zone.now + 2.days
 2.days.from_now
 ----
 
+Use an instance of `ActiveSupport::Duration` as a receiver to calculate relative time.
+
+[source,ruby]
+----
+# bad - It's confusing to read
+created_at.since(1.minute)
+expiration_date.ago(3.days)
+
+# good
+1.minute.since(created_at)
+3.days.ago(expiration_date)
```

Comment:
According to a guideline above

> If used with a parameter, prefer `since`, `after`, `until` or `before` instead of `from_now` and `ago`.

`3.days.ago(expiration_date)` would be wrong, too.

And correct options do not map to `ago`/`in`, too.

WDYT of only keeping the `since` example here and in the "bad" section?